vcpkg/concepts/offline.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Offline Usage of vcpkg
+description: Use export, asset caching, and git to develop in offline or air-gapped networks.
+author: ras0219-msft
+ms.author: roschuma
+ms.topic: concept-article
+ms.date: 03/12/2025
+---
+# Offline Usage of vcpkg
+
+This document details two robust methods for deploying vcpkg in environments without live internet connectivity.
+
+## Method 1: Prebuild and Export Packages
+
+This method uses the export command to create a comprehensive package bundle on an internet-connected system. The process compiles binaries, headers, and all transitive dependencies into a self-contained SDK bundle suitable for offline deployment.
+
+Get started with the [Export Tutorial – Creating a self-contained SDK bundle](../produce/export-compiled-dependencies.md).
+
+### Pros and Cons
+
+**Pros:**
+- All necessary components are bundled together into a single archive using the export command.
+- Easy to get started.
+- Extremely reliable.
+
+**Cons:**
+- Updates and source development must be performed on a connected system.
+- Involves additional maintenance due to frequent updates.
```

Comment:
I don't completely follow this point. Why would there be frequent updates? What is being updated?

Is this for the development scenario, where the users are packaging a dependency rather than just consuming?

Reply:
Updated!